### DISCUSSION TITLE
Toga for Game Devlopment?

### DISCUSSION THREAD

**Discussion by ghost** [beeware/toga#1511] [Ideas]
In the first place, can i manipulate every (well almost) pixel? If I can, what are the pros and cons of this. 
(i hope i put this in the right category)

**Comment by freakboy3742** [comment]
Toga isn't really a good match for game development. Toga's focus is on "desktop"-style apps - apps with buttons, text inputs, sliders, radio buttons, and so on, rendered using the operating system's default style. Game development requires optimised bitmap rendering with support for sprites; or 3D canvases with OpenGL (or similar) APIs.  

While you certainly *could* write a game with Toga, you'll make a lot more progress using a framework that is focussed on games as a target.
